bindings/kotlin/example/build.gradle

```diff
@@ -38,5 +44,12 @@ dependencies {
     implementation 'androidx.compose.material3:material3'
     implementation "androidx.activity:activity-compose:1.7.2"
 
-    implementation project(':lib')
+    if (useLocalLib) {
+        implementation project(':lib')
+    }
+    else {
+        implementation 'ie.equalit.ouinet:ouisync-omni:0.7.2'
+        implementation 'net.java.dev.jna:jna:5.13.0@aar'
+        implementation 'org.msgpack:msgpack-core:0.9.5'
```

Comment:
Why are these dependencies (jna and msgpack) repeated here? Aren't they transitively included through the lib?

Reply:
No, not by default. The AAR will not work with them added here. I believe that is is possible to include them in the AAR, maybe adding them to the pom file? I can look into it a little. 

Though this might not be preferable, since including them _might_ cause conflicts if someone wanted to use the ouisync AAR and a different version of jna or msgpack. But this issue can be worked around with an exclude statement, i think.

We have a similar situation with `relinker` for ouinet, it must be added as dep in addition to the ouinet aar. Maybe this was a conscious choice by @mhqz?

Reply:
Ok, no worries. I'm not that familiar with how things are done in the java world so maybe this is ok. Thanks, I'll merge.

Reply:
Thanks for the merge. Note that I made a typo in my original response and just edited it (Now it reads correctly, "The AAR will not work **without** them added here")

And my investigation in to including them as transitive dependencies went nowhere. It is probably possible, but I can't figure out how to get gradle/sonatype/maven to do it.